src/shipit_pulse_listener/shipit_pulse_listener/listener.py

```diff
@@ -255,23 +256,27 @@ def maybe_trigger(tasks):
 
             return None
 
-        list_url = 'https://queue.taskcluster.net/v1/task-group/' + group_id + '/list'
-
-        r = requests.get(list_url, params={
-            'limit': 200
-        })
-        reply = r.json()
-        task = maybe_trigger(reply['tasks'])
-
-        while task is None and 'continuationToken' in reply:
-            r = requests.get(list_url, params={
-                'limit': 200,
-                'continuationToken': reply['continuationToken']
+        def retrieve_coverage_task():
+            r = requests.get('https://queue.taskcluster.net/v1/task-group/{}/list'.format(group_id), params={
+                'limit': 200
             })
             reply = r.json()
             task = maybe_trigger(reply['tasks'])
 
-        return task
+            while task is None and 'continuationToken' in reply:
+                r = requests.get(list_url, params={
```

Comment:
Where does `list_url` comes from ?

Reply:
It's defined a few lines earlier: `list_url = 'https://queue.taskcluster.net/v1/task-group/{}/list'.format(group_id)`

Reply:
Okay, i missed it by reading an outdated commit (8a6f8ef7a97249544cc40e300d34567c1a4a847e)